src/main/java/codesquad/codestagram/controller/UserController.java

```diff
@@ -0,0 +1,64 @@
+package codesquad.codestagram.controller;
+
+import codesquad.codestagram.domain.User;
+import codesquad.codestagram.repository.MemoryUserRepository;
+import codesquad.codestagram.repository.UserRepository;
+import codesquad.codestagram.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+//UserController를 추가하고 @Controller 애노테이션 추가
+@Controller
+@RequestMapping("/users")
+public class UserController { //url을 읽어서 처리
+
+    private final UserService userService;
+
+    private final MemoryUserRepository memoryUserRepository;
```

Comment:
```suggestion
    private final UserRepository userRepository;
```

SOLID 원칙에 대해 다시 확인해 보세요.